pytest_html/plugin.py

```diff
@@ -172,9 +184,8 @@ def append_extra_html(self, extra, extra_index, test_index):
                     href = src = self.create_asset(
                         content, extra_index, test_index,
                         extra.get('extension'), 'wb')
-                    self.additional_html.append(html.div(
-                        html.a(html.img(src=src), href=href),
-                        class_='image'))
+                self.images_html.append(html.a(html.img(
+                    src=src), href=href, target='_blank'))
```

Comment:
In the case that the `src` is a base64 encoded string, we're duplicating it for the `href`. This will cause considerable increase in the file size if there's more than a few screenshots. This is actually related to https://github.com/pytest-dev/pytest-html/issues/110, where I suggest resolving this using JavaScript.